website/docs/getting-started.md

```diff
@@ -33,14 +33,14 @@ See the detailed [Docs on how to use the ConfigCat SDKs.](/sdk-reference/overvie
 Here's a short example to demonstrate the concept:
 
 ```js
-var configcat = require('configcat-client');
-var client = configcat.createClient('YOUR SDK KEY HERE');
-
-client.getValue('isMyFeatureEnabled', false, (value) => {
-  if (value === true) {
-    do_the_new_thing();
-  } else {
-    do_the_old_thing();
-  }
-});
+const configcat = await import("https://cdn.jsdelivr.net/npm/configcat-js/+esm");
```

Comment:
Nekem jobban tetszett az előző irány, ez nekem valahogy gagyinak hat egy getting startedben. Legtöbb esetben már szerencsére nem az a use case, hogy így használják.
Ha még egy npm i configcat-js code blokk is lenne előtte, és maradna a require vagy 
import * as configcat from 'configcat-js';
Nekem jobban tetszene.

Reply:
Az volt a gondolat, hogy ezt egy az egyben be lehet copyzni a böngészőbe vagy valami js scratchpadbe és ki lehet próbálni.

Az `import * as configcat from 'configcat-js';` típusú megoldáshoz már telepíteni kell legalább egy node-ot.

Nekem jó ez is, viszont akkor require legyen és `from 'configcat-node'`, mert egyébként még valami bundler is fel kell konfigolni, ha valaki le akarja tesztelni.

Reply:
BTW, találtam egy egyszerűbb URL-t: "https://esm.sh/configcat-js". Ezt használja a codepen.io is.

Reply:
Értem, hogy mi volt az alapgondolat, de attól még én ebben a code example-ben egyáltalán nem szeretnék ilyen url-eket látni. A legtöbb esetben nem így van használva az SDK, és a getting started-ben nem egy olyan kóddal szeretnék nyitni, ami egy edge usage case. Az, hogy telepíteni kell valamilyen node-ot, szerintem nem probléma, mert az az általános használat, és szinte mindenki úgy használja az SDK-t.
Lehet érdemes lenne beszélni róla reggeli meetingen többiekkel.

Reply:
A node-os példa mégse túl jó ötlet, mert a node-ban nincs top level await (legalábbis ha require-t használunk). Úgyhogy ezt bírtam kiötleni: https://github.com/configcat/docs/pull/348/commits/79a569a5b776eac0cc0a77614d00d04e5f844e56

Ez hogy tetszik?

Reply:
Én személy szerint az or-os commentet nem tenném bele, szerintem elég anélkül is. De ha nagyon szeretnéd így, nekem okés.

Reply:
Nem ragaszkodom hozzá, max. ilyen nice-to-have dolognak érzem. Törlöm.